toolkit/javascripts/live-search.js

```diff
@@ -102,6 +103,14 @@ endpoint response (application/json):
     }
   };
 
+  LiveSearch.prototype.filterSelected = function(){
+    if(this.checked) {
+      var key = $(this).attr('name');
+      var value = $(this).attr('value');
+      GOVUK.GDM.analytics.virtualPageViews.trackFilterAnalytics(key,value);
```

Comment:
I think that's a valid point. We should probably implement this on the toolkit as it can be used generically.